src/bundler.ts

```diff
@@ -41,7 +41,7 @@ export class Bundler {
 
     public async BundleAll(
         files: string[],
-        dedupeGlobs: string[]
+        dedupeGlobs?: string[]

```

Comment:
Actually, this should default to an empty string array, if you want to make it optional:
```
dedupeGlobs: string[] = []
```